src/videojs-http-streaming.js

```diff
@@ -883,5 +915,6 @@ export {
   HlsHandler,
   HlsSourceHandler,
   emeKeySystems,
-  simpleTypeFromSourceType
+  simpleTypeFromSourceType,
+  expandDataUri
```

Comment:
is this something that's needed to be exposed?

Reply:
Just for tests